glib/src/gstring.rs

```diff
@@ -640,12 +1128,6 @@ impl AsRef<str> for GString {
     }
 }
 
-impl AsRef<CStr> for GString {
```

Comment:
Why?

Reply:
Replaced with some `to_cstr` functions since `GStr` can now contain interior nuls and `AsRef` implementations should probably not re-scan the string or panic

Reply:
ack